java/src/main/java/com/saucelabs/simplesauce/SauceOptions.java

```diff
@@ -3,24 +3,78 @@
 import com.saucelabs.simplesauce.enums.MacVersion;
 import lombok.Getter;
 import lombok.Setter;
-import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.MutableCapabilities;
 import org.openqa.selenium.remote.BrowserType;
+import org.openqa.selenium.remote.CapabilityType;
+
+import java.util.HashMap;
 
 public class SauceOptions {
+    @Getter private MutableCapabilities seleniumCapabilities;
+
     @Getter @Setter private String browserName = BrowserType.CHROME;
     @Getter @Setter private String browserVersion = "latest";
-    @Getter @Setter private String operatingSystem = Platforms.windowsLatest().getOsVersion();
-    @Getter private ChromeOptions chromeOptions;
+    @Getter @Setter private String platformName = Platforms.windowsLatest().getOsVersion();
+    @Getter private final SauceTimeout sauceTimeout = new SauceTimeout();
+    @Setter private String build;
+
+    public SauceOptions() {
+        this(new MutableCapabilities());
+    }
+
+    public SauceOptions(Capabilities capabilities) {
+        seleniumCapabilities = new MutableCapabilities(capabilities);
+        if (capabilities.getCapability("browserName") != null) {
+            browserName = (String) capabilities.getCapability("browserName");
+        }
+    }
+
+    public MutableCapabilities toCapabilities() {
+        seleniumCapabilities.setCapability(CapabilityType.BROWSER_NAME, browserName);
+        seleniumCapabilities.setCapability(CapabilityType.PLATFORM_NAME, platformName);
+        seleniumCapabilities.setCapability(CapabilityType.BROWSER_VERSION, browserVersion);
+        seleniumCapabilities.setCapability("sauce:options", new HashMap<>());
+        return seleniumCapabilities;
+    }
+
+    public String getBuild() {
```

Comment:
The thought was to keep all of the field names the same as what is in the sauce documentation.

Reply:
let's decide what these need to look like in the accessors PR. It might not make sense for build to be "special" in the first place if we go for something more generic.